src/App.js

```diff
@@ -108,21 +102,19 @@ function App() {
       });
   }
 
-  useEffect(() => {
-    fetch(`http://localhost:3000/days`)
-      .then((r) => r.json())
-      .then((data) =>
-        setCurrentDaysData(data.filter((dayObj) => dayObj.user_id === 1))
-      );
-  }, [rerender]);
+  const fetchData = (url, dependencyArray) => {
+    return () => {
+      fetch(url)
+        .then((r) => r.json())
+        .then((data) => setCurrentDaysData(data.filter((dayObj) => dayObj.user_id === 1)));
+    };
+  };
 
-  useEffect(() => {
-    fetch(`http://localhost:3000/days`)
-      .then((r) => r.json())
-      .then((data) =>
-        setCurrentDaysData(data.filter((dayObj) => dayObj.user_id === 1))
-      );
-  }, [activateRerender]);
+const fetchDataForRerender = fetchData("http://localhost:3000/days", [rerender]);
+const fetchDataForActivateRerender = fetchData("http://localhost:3000/days", [activateRerender]);
+
+useEffect(fetchDataForRerender, [rerender]);
+useEffect(fetchDataForActivateRerender, [activateRerender]);
```

Comment:
`question`: Why are there multiple `useEffect` invocations here? Have you read the React documentation or community forums about whether this is a best practice or not? Why or why not would this be a best practice?